crates/ruma-common/src/events/relation.rs

```diff
@@ -182,8 +178,7 @@ pub enum RelationType {
     Replacement,
 
     /// `m.thread`, a participant to a thread.
-    #[cfg(feature = "unstable-msc3440")]
-    #[ruma_enum(rename = "io.element.thread", alias = "m.thread")]
+    #[ruma_enum(rename = "m.thread", alias = "io.element.thread")]
```

Comment:
Same here

Reply:
This one is less problematic since it's used only inside a URI's path, so a client cannot send both, but let's remove it anyway.